lib/OpenQA/Schema/Result/Jobs.pm

```diff
@@ -360,16 +339,12 @@ sub reschedule_state {
     log_debug('Job ' . $self->id . " reset to state $state");
 
     # free the worker
-    if (my $worker = $self->worker) {
-        $self->worker->update({job_id => undef});
-    }
+    if (my $worker = $self->worker) { $self->worker->update({job_id => undef}) }
```

Comment:
This looks like it was already intended to do this in the original code:
```suggestion
    if (my $worker = $self->worker) { $worker->update({job_id => undef}) }
```

Reply:
done